README.md

```diff
@@ -1,2 +1,37 @@
 # To-Do-List
  This is a simple "To-do list"  project that helps to organize your day built using Webpack.
+
+ # Built With
+   - Html
+   - Css 
+   - JavaScript and 
+   - Webpack
+
+# Getting Started
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites: 
+Any kind of browser. 
+
+### Setup:
+    1. Clone the repository (git clone https://github.com/Ney2/Webpack-Setup.git)
+    2. Then open the folder in your local environment 
+    3. Choose the file with a name index.html file
+    4. Run index.html file using any browser in your local environment
+
```

Comment:
- [ ] [OPTIONAL] Your readme file looks descriptive and professional but please Update your readme file with explicit steps on how to clone and run your tests locally, as this will be of help to non-technical users. :smile:

Reply:
Okay, I'll correct it. Thank you for your suggestions and for reviewing my project.